src/vs/workbench/api/node/extHost.protocol.ts

```diff
@@ -257,6 +264,12 @@ export abstract class ExtHostEditorsShape {
 	$acceptTextEditorRemove(id: string): void { throw ni(); }
 }
 
+export abstract class ExtHostTreeExplorersShape {
```

Comment:
Oops sorry. Actually I designed with this in mind and should use `InternalTreeExplorerNodeContent`, which is an interface.